api_app/analyzers_manager/migrations/0094_analyzer_config_detectiteasy.py

```diff
@@ -0,0 +1,124 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "detectiteasy.DetectItEasy",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "DetectItEasy",
+    "description": "[DetectItEasy](https://github.com/horsicq/Detect-It-Easy) is a program for determining types of files.",
+    "disabled": False,
+    "soft_time_limit": 60,
```

Comment:
maybe a little less is enough, I expect that this tool would execute for a very short time (max 3 seconds). What is your practical experience with the files you tried?

Reply:
![image](https://github.com/intelowlproject/IntelOwl/assets/102874321/a2423e4c-2a66-4cd6-a63e-5935487cad85)
this scan took 7 seconds. Another scan mentioned [here](https://github.com/intelowlproject/IntelOwl/pull/2354#issuecomment-2145985443) takes 3s. 
should we keep it 10 then?

Reply:
yeah let's try with that